components/ui/command.tsx

```diff
@@ -0,0 +1,155 @@
+"use client"
+
+import * as React from "react"
+import { DialogProps } from "@radix-ui/react-dialog"
+import { Command as CommandPrimitive } from "cmdk"
+import { Search } from "lucide-react"
```

Comment:
Remove `lucide-react`, use search icon from heroicons and place into icons file. (aside: unoptimized lucide is terrible for performance)

Reply:
OK I used this shadcn cli to create component, I will refactor this :)